README.md

```diff
@@ -131,6 +131,14 @@ application's public ID.
 <script src="https://cdn.croct.io/js/v1/lib/plug.js?appId=<APP_ID>"></script>
 ```
 
+### Function call
```

Comment:
You need dedicated sections for each installation method as they are not the same.

Suggestion:

### NPM

The recommended way to install the SDK is via [NPM](https://npmjs.com). It pairs nicely with module bundlers such as Webpack or Browserify and includes Typescript typings.

In most cases, it should be as simple as running the following in your project:

```sh
npm install @croct/plug
```

Then, call [`croct.plug`](docs/plug.md#plug) passing the App ID to  initialize the SDK:

```js
import croct from '@croct/plug';

croct.plug({appId: '<APP_ID>'});
```
### Script Tag

To install the SDK as a script tag, add the following line to the `<head>` tag of your site on any pages you plan to use the SDK to personalize or track events:

```html
<script src="https://cdn.croct.io/js/v1/lib/plug.js?appId=<APP_ID>"></script>
<script>croct.plug();</script>
```

You should replace the `<APP_ID>` placeholder with the respective App ID.  For more information about the available options, see [`croct.plug`](docs/plug.md#plug).